frontend/packages/ui/src/ProgressBar.tsx

```diff
@@ -0,0 +1,144 @@
+import React from 'react'
+import { useState, useEffect } from 'react'
+import { BarChart, Bar, XAxis, YAxis, Tooltip, ResponsiveContainer } from 'recharts'
+import { Colors } from './styles/colors'
+import styled from 'styled-components'
+
+interface Props {
+  data: { 
+    name?: string;
```

Comment:
have only percentage and name. And we will handle that outside of the component.

Reply:
Done